shared/components/SpeakersList/index.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/no-danger */
+
+import React, { PropTypes } from 'react';
+import speakerType from '../../prop-types/speaker-type';
+import SpeakerSocialLinks from '../SpeakerSocialLinks';
+
+export const getSpeakerAvatarImage = (imageURL, speakerName) => {
+  if (imageURL) {
+    return (<img
+      alt={`Speaker ${speakerName}`}
+      src={imageURL}
+    />);
+  }
+  return (
+    <div className="icon-default-speaker-conference-avatar" />
+  );
+};
+
+const SpeakersList = ({ speakers }) => {
+  if (speakers && Array.isArray(speakers) && speakers.length) {
+    const speakerList = speakers.map((speaker) => {
+      return (
+        <li key={speaker.id}>
+          <div className="speaker-avatar">
+            {getSpeakerAvatarImage(speaker.imageURL, speaker.name)}
+          </div>
+          <div className="speaker-description">
+            <div className="speaker-description-name">{speaker.name}</div>
+            <div className="speaker-description-company">{speaker.company}</div>
```

Comment:
Wrapped into `h3` and changed page header to `h2` same way as it is done on Community page.